lib/Validate.ts

```diff
@@ -1,8 +1,41 @@
+export enum Strategies {
+  SkipValidation = 'skip-validation',
```

Comment:
Do we need this one?
Since this is basically a no-op, perhaps it's clearer if users just don't use this lib if they don't need validation.

Reply:
good idea